test/logflare/billing/billing_counts_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Logflare.Billing.BillingCountsTest do
+  use Logflare.DataCase
+  alias Logflare.Billing.BillingCounts
+
+  setup do
+    user = insert(:user)
+
+    _non_matching_user = insert(:billing_counts, inserted_at: ~U[2023-02-01 00:00:00Z])
+
+    _outside_boundary = insert(:billing_counts, user: user, inserted_at: ~U[2023-01-31 00:00:00Z])
+
+    _outside_boundary = insert(:billing_counts, user: user, inserted_at: ~U[2023-03-01 00:00:00Z])
+
+    inside_boundary_0 = insert(:billing_counts, user: user, inserted_at: ~U[2023-02-17 00:00:00Z])
+
+    inside_boundary_1 = insert(:billing_counts, user: user, inserted_at: ~U[2023-02-17 00:00:00Z])
+
+    inside_boundary_2 = insert(:billing_counts, user: user, inserted_at: ~U[2023-02-10 00:00:00Z])
+
```

Comment:
a little more verbose than necessary, 2 in boundary and 1 outside boundary would suffice.

Reply:
I want to test the cumulative nature of the time-series output

Reply:
👌👌